src/scribe_data/utils.py

```diff
@@ -420,3 +420,21 @@ def check_and_return_command_line_args(
         python {all_args[0]} '["comma_separated_sets_in_quotes"]'
         """
     )
+
+def map_genders(wikidata_gender):
```

Comment:
If I'm understanding https://github.com/scribe-org/Scribe-Data/issues/69 correctly, it seems that what we'd rather want here actually is that `map_genders()` also works for all supported languages. There are other `map_genders()` functions for other languages that exist elsewhere in the code, as pointed out in the issue. The changes here only address the `map_genders()` for German.

So to make the function general and reconcile all supported languages, it could make sense to modify `map_genders(wikidata_gender)` into something like `map_genders(wikidata_gender, iso_code)` instead perhaps - where `iso_code` is `"en"`, `"fr"`, `"es"`, etc. We can then use the `iso_code` for the language-specific checks, such as `"Q499327"` here  for German masculine.

Let me know if there are any questions regarding the above :smile:

Reply:
Thank you so much for the review, it was helpful in understanding the codebase. I can see that we have `map_genders()` function for other languages.

I have tried implementing this feedback however on running `update.py` file, I got a couple of files modified and a new file created(`src/scribe_data/extract_transform/languages/German/nouns/nouns_queried.json`).

Can I commit these extra files?

Reply:
Let's not commit the extra files like `nouns_queried.json`, @ikeadeoyin :) These should have been deleted by the `format_nouns.py` file in the German directory. The way it works is that the query results are brought down, they're formatted and saved in the final version, and then the original file is deleted. Let's just have this focus on the Python files 😊

Reply:
Alright @andrewtavis,

Following the contribution guide, I have committed the Python files however, I have this error
```
Scribe-Data git:(add-map-gender-function) ✗ git pull --rebase upstream main
error: cannot pull with rebase: You have unstaged changes.
error: Please commit or stash them. 
```

Reply:
Hey @ikeadeoyin :wave: 
Are the unstaged changes the `*.json` files?
If so, I guess you could either delete them from your local machine or run `git stash` to [stash](https://www.atlassian.com/git/tutorials/saving-changes/git-stash#stashing-your-work) them so that your working directory is clean before pulling